docs/compliance/index.md

```diff
@@ -0,0 +1,386 @@
+.\1-DeployAndConfigureAzureResources.ps1 
+    -resourceGroupName contosowebstore
+    -globalAdminUserName adminXX@contosowebstore.onmicrosoft.com 
+    -globalAdminPassword **************
+    -azureADDomainName contosowebstore.onmicrosoft.com 
+    -subscriptionID XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX 
+    -suffix PCIcontosowebstore
+    -customHostName contosowebstore.com
+    -sqlTDAlertEmailAddress edna@contosowebstore.com 
+    -enableSSL
+    -enableADDomainPasswordPolicy 
+```
+
+For detailed usage instructions, see [Script Instructions - Deploy and Configure Azure Resources](./1-DeployAndConfigureAzureResources.md).
+
+3. Deploy OMS logging and resources
+ ```powershell
+.\2-EnableOMSLoggingOnResources.ps1 
+    -resourceGroupName contosowebstore 
+    -globalAdminUserName adminXX@contosowebstore.onmicrosoft.com 
+    -globalAdminPassword **************
+    -subscriptionID XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX
+```
+    
+For detailed usage instructions, see [Script Instructions - Payment Sample Dataset](./pci-sample-dataset.md). 
+
+## User Scenario
+
+The foundational architecture addresses the use case below.
+
+> This scenario illustrates how a fictitious webstore moved their payment card processing to a Azure-based solution. The solution handles collection of basic user information including payment data. The solution does not process payments with this cardholder data; once the data is collected, customers are responsible for initiating and completing transactions with a payment processor. For more information, see the "Review and Guidance for Implementation" document at the [Microsoft Service Trust Portal](http://aka.ms/stp).
+
+### Use Case
+A small webstore called *Contoso Webstore* is ready to move their payment system to the cloud. They have selected Microsoft Azure to host the process for purchasing and to allow a clerk to collect credit card payments from their customers.
+
+The administrator is looking for a solution that can be quickly deployed to achieve his goals in for a cloud-born solution. He will use this proof-of-concept (POC) to discuss with his stakeholders how Azure can be used to collect, store, and retrieve payment card data while complying with stringent Payment Card Industry Data Security Standard (PCI DSS) requirements.
+
+> You will be responsible for conducting appropriate security and compliance reviews of any solution built with the architecture used by this POC, as requirements may vary based on the specifics of your implementation and geography. PCI DSS requires that you work directly with an accredited Qualified Security Assessor to certify your production-ready solution.
+
+### Elements of the Foundational Architecture
+
+The foundational architecture is designed with the following fictitious elements:
+
+Domain site `contosowebstore.com`
+
+User roles used to illustrate the use case, and provide insight into the user interface.
+
+#### Role: Site and Subscription Admin
+
+|Item      |Example|
```

Comment:
Do we need these tables? I think readers will care about the role descriptions (below) but not the usernames etc